### PR TITLE
Update Tiebreaker.isTie() to match the mechanics doc

### PR DESCRIPTION
**Pull request by Psirex**
Add the additional condition **"protocol withdrawals are paused for a duration exceeding `TiebreakerActivationTimeout`"** to check whether the sealable withdrawal blocker is paused.

### CODE REVIEW

File: test/unit/libraries/Tiebreaker.t.sol [hunk 1]
```diff
@@ -303,12 +307,35 @@ contract TiebreakerTest is UnitTest {
...
+
+        // simulate tiebreaker was locked for time < tiebreakerActivationTimeout
+        _mockSealableResumeSinceTimestampResult(_SEALABLE, block.timestamp + 14 days);
+
+        // then tiebreaker activation is not allowed
+        vm.expectRevert(Tiebreaker.TiebreakNotAllowed.selector);
+        this.external__checkTie(DualGovernanceState.VetoSignallingDeactivation, normalOrVetoCooldownExitedAt);
+
+        // check edge case when resumeSinceTimestamp == block.timestamp + tiebreakerActivationTimeout
+        _mockSealableResumeSinceTimestampResult(_SEALABLE, block.timestamp + tiebreakerActivationTimeout.toSeconds());
```
Comment (Psirex) [inline test/unit/libraries/Tiebreaker.t.sol:322]: Consider edge cases

File: contracts/libraries/Tiebreaker.sol [hunk 2]
```diff
@@ -171,17 +172,31 @@ library Tiebreaker {
...
+            self.tiebreakerActivationTimeout.addTo(Timestamps.now()).toSeconds();
+
         for (uint256 i = 0; i < sealableWithdrawalBlockersCount; ++i) {
-            (bool isCallSucceed, bool isPaused) = SealableCalls.callIsPaused(self.sealableWithdrawalBlockers.at(i));
-            if (isPaused || !isCallSucceed) return true;
+            (bool isCallSucceed, uint256 resumeSinceTimestampInSeconds) =
+                SealableCalls.callGetResumeSinceTimestamp(self.sealableWithdrawalBlockers.at(i));
+            /// @dev If the call failed, mark the sealable as faulty.
+            ///     Otherwise, consider it paused if its pause duration exceeds `tiebreakerActivationTimeout`.
+            if (!isCallSucceed || tiebreakAllowedTillTimestampInSeconds <= resumeSinceTimestampInSeconds) {
```
Comment (Psirex) [inline contracts/libraries/Tiebreaker.sol:197]: Seems like more correct to have `<` instead of `<=` here

File: contracts/libraries/Tiebreaker.sol [hunk 3]
```diff
@@ -67,7 +67,8 @@ library Tiebreaker {
         if (sealableWithdrawalBlockersCount == maxSealableWithdrawalBlockersCount) {
             revert SealableWithdrawalBlockersLimitReached();
         }
-        (bool isCallSucceed, /* isPaused */ ) = SealableCalls.callIsPaused(sealableWithdrawalBlocker);
+        (bool isCallSucceed, /* resumeSinceTimestamp */ ) =
+            SealableCalls.callGetResumeSinceTimestamp(sealableWithdrawalBlocker);
         if (!isCallSucceed) {
             revert InvalidSealable(sealableWithdrawalBlocker);
         }
```
Comment (Psirex) [inline contracts/libraries/Tiebreaker.sol:74]: if (resumeSinceTimestamp >= block.timestamp) {
  revert()
}
